src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/jpa/repository/LocationMappingRepository.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.repository
+
+import kotlinx.coroutines.flow.Flow
+import org.springframework.data.domain.Pageable
+import org.springframework.data.repository.kotlin.CoroutineCrudRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.LocationMapping
+import uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.LocationMappingType
+
+@Repository
+interface LocationMappingRepository : CoroutineCrudRepository<LocationMapping, Long> {
+  suspend fun findOneByNomisLocationId(nomisLocationId: Long): LocationMapping?
+  suspend fun findFirstByMappingTypeOrderByWhenCreatedDesc(mappingType: LocationMappingType): LocationMapping?
+  suspend fun countAllByLabelAndMappingType(label: String, mappingType: LocationMappingType): Long
+  fun findAllByLabelAndMappingTypeOrderByLabelDesc(label: String, mappingType: LocationMappingType, pageable: Pageable): Flow<LocationMapping>
```

Comment:
why is this returning a flow? Can't it just be a `suspend` and return a `List<LocationMapping>`?

Reply:
All the repos do this (otherwise I get an error: "Method has to use a either multi-item reactive wrapper return type or a wrapped Page/Slice type")